src/kernel/kmain.c

```diff
@@ -134,6 +139,17 @@ void kmain(uint64_t addr)
     print_ko();
   }
 
+  print_step("loading kernel system font");
+
+  kernel_console_font = psf1_load_font("/fonts/zap-light16.psf");
```

Comment:
The license for this font does not seem compatible with this project. We should try to find one licensed under SIL Open Font License.

Reply:
Updated. It's not the best font but we can probably spend some more time later to get a better one.